addons/website_event/views/event_templates_list.xml

```diff
@@ -149,9 +149,20 @@
     <!-- No events -->
     <t t-if="not event_ids">
         <div class="col-12">
-            <div class="h2 mb-3">No events found.</div>
-            <div class="alert alert-info text-center" groups="event.group_event_user">
-                <p class="m-0">Use the top button '<b>+ New</b>' to create an event.</p>
+            <div class="d-flex flex-column align-items-center justify-content-center text-center">
+                <div class="mb-3 o_event_svg">
```

Comment:
`o_event_ ` prefix : In other files you added "website" -> `o_wevent_` .
Both are good but pick one, preferably the one already used by the modules ;)

Classes order: custom classes first, then utility classes.